tests/models/xapi/base/test_statements.py

```diff
@@ -500,7 +501,9 @@ def test_models_xapi_base_statement_with_valid_version(statement):
         # We have to bypass Video and Virtual Classroom Statements in
         # this test because we want to support
         # invalid values (non IRI keys) in their extension fields.
-        if not issubclass(model, (BaseVideoStatement, BaseVirtualClassroomStatement))
+        if not issubclass(
+            model, (BaseVideoStatement, BaseVirtualClassroomStatement, BaseLMSStatement)
+        )
```

Comment:
```suggestion
        if not issubclass(
            model, (BaseVideoStatement, BaseVirtualClassroomStatement, LMSDownloadedVideo)
        )
```
It seems that only the `LMSDownloadedVideo` statement is not a valid `BaseXapiStatement`.
Could we adapt this test to work with other `BaseLMSStatement`s?

For example we could change the following line in the test:
`statement = data.draw(custom_builds(model)).dict(exclude_none=True)`
with:
`statement = data.draw(custom_builds(model)).json(exclude_none=True, by_alias=True)`

and instead of:
`BaseXapiStatement(**statement)`
use:
`BaseXapiStatement(**json.loads(statement))`

Reply:
Also registration models would fail this test
I have updated the test as you proposed